operations/helm/charts/grafana-agent/templates/controllers/_pod.yaml

```diff
@@ -19,6 +19,10 @@ spec:
     {{- toYaml .Values.image.pullSecrets | nindent 4 }}
     {{- end }}
   {{- end }}
+  initContainers:
```

Comment:
can we include this in a conditional, so that the field doesn't exist if there are no containers. It is a bit cosmetic, but I think we'd rather the manifests not have empty fields.

Reply:
flow control has been added